README.md

```diff
@@ -100,11 +108,30 @@ evaluate(
 
 Performs the given `operation` on `brushA` with `brushB`. If no target is provided then a new `Brush` will be created with the new geometry. Otherwise the provided Brush will be _modified in place_ and geometry disposed or marked for update as needed.
 
-<!--
+
 ### .evaluateHierarchy
 
-TODO
+```js
+evaluateHierarchy( 
+  root: Opertation, 
+  target = new Brush : Brush | Mesh
+) : Brush | Mesh
+```
+
+The method gets as parameters a root, an Operation mesh and a target if it is provided, otherwise, a new `Brush` will be created. 
+First sets the `updateMatrixWolrd` of the root to true then calls the traverse function with the root parameter to `evaluate` the mesh and its children. 
+
+```js
+evaluate( brush, child, child.operation );
+```
+
+In this case, the arguments passed to `evaluate` is the `root` as `brushA`, the child as `brushB` and the `child.operation` as the operation to apply to the mesh.
+
+## OperationDebugData
```

Comment:
Lets at least document `enabled` and `intersectionEdges` for this class. I can fill in some of the content if it's not clear what it does.